src/org/usfirst/frc/team4571/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,64 @@
+package org.usfirst.frc.team4571.robot.subsystems;
+
+import java.util.logging.Logger;
+
+import org.usfirst.frc.team4571.robot.RobotConstants;
+import org.usfirst.frc.team4571.robot.subsystems.PID.DistanceOutput;
+import org.usfirst.frc.team4571.robot.subsystems.sensors.EncoderAverage;
+
+import com.ctre.CANTalon;
+
+import edu.wpi.first.wpilibj.CounterBase.EncodingType;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.wpilibj.RobotDrive;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class IntakeSubsystem extends Subsystem {
+	private CANTalon rightMotor;
+	private boolean isRollerIn;
+	public static final DoubleSolenoid ROLLER_SOLENOID = new DoubleSolenoid(RobotConstants.ROLLER_FOWARD_SOLENOID_CHANNEL, RobotConstants.ROLLER_REVERSE_SOLENOID_CHANNEL);
```

Comment:
May we initialize ROLLER_SOLENOID the same way rightMotor is being initialized right now?
Both initialize WPILib controls and should maintain consistency.